test/resources/V1_2/LccRoundTripRef.xml

```diff
@@ -5,8 +5,8 @@
             <iidm:busBreakerTopology>
                 <iidm:bus id="B1"/>
             </iidm:busBreakerTopology>
-            <iidm:shunt id="C1_Filter1" name="Filter 1" bPerSection="1.0000000000000001e-05" maximumSectionCount="1" currentSectionCount="1" bus="B1" connectableBus="B1" q="25"/>
-            <iidm:shunt id="C1_Filter2" name="Filter 2" bPerSection="2.0000000000000002e-05" maximumSectionCount="1" currentSectionCount="0" connectableBus="B1" q="25"/>
+            <iidm:shunt id="C1_Filter1" name="Filter 1" bPerSection="1.0000000000000001e-05" maximumSectionCount="1" currentSectionCount="1" voltageRegulatorOn="false" bus="B1" connectableBus="B1" q="25"/>
```

Comment:
It looks like it was a bug in ShuntCompensatorXml in v1.2... The XML is now the same as in Java (`voltageRegulatorOn` attribute is emitted, whether it is true or false)

Reply:
We need to ensure the bug is in the v1.2 and add this information in the release note as a known bug

Reply:
I confirm it is a bug in v1.2 : 
C++ v1.2 : https://github.com/powsybl/powsybl-iidm4cpp/blob/integration/v1.2.0/src/iidm/converter/xml/ShuntCompensatorXml.cpp
Java 1.2 : https://github.com/powsybl/powsybl-core/blob/evolution_xiidm/1.2/iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/ShuntXml.java